src/libraries/Common/tests/System/Security/Cryptography/X509Certificates/X509CertificateLoaderPkcs12CollectionTests.cs

```diff
@@ -178,7 +178,7 @@ public abstract partial class X509CertificateLoaderPkcs12CollectionTests
 #if NETFRAMEWORK
             X509KeyStorageFlags.DefaultKeySet;
 #else
-           PlatformDetection.UsesAppleCrypto ? 
+           PlatformDetection.UsesAppleCrypto ?
```

Comment:
Sorry for the white-space changes but my editor _really_ wants to remove them, and according to our .editorconfig rules they should be trimmed.

https://github.com/dotnet/runtime/blob/c20e5e19e0f262ba5a9546a5786e3d98982c82d1/.editorconfig#L13